modules/install-creating-install-config-aws-local-zones-subnets.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="install-creating-install-config-aws-local-zones-subnets_{context}"]
+= Modifying an installation configuration file to use AWS Local Zones subnets
+
+Modify an `install-config.yaml` file to include AWS Local Zones subnets.
+
+.Prerequisites
+
+* You created subnets by using the procedure "Creating a subnet in AWS Local Zones".
+* You created an `install-config.yaml` file by using the procedure "Creating the installation configuration file".
+
+.Procedure
+
+* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property. As an example:
```

Comment:
```suggestion
* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property, as in the following example:
```

Reply:
A complete sentence is needed for introductory text: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons#introductory-text